src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -134,7 +134,7 @@ export const SearchResult = ({ school, schoolIDs }) => (
       <div className="vads-l-col--6 vads-u-padding-left--2">
         {/* School Website */}
         <h4 className="vads-u-font-family--sans vads-u-font-size--h5 vads-u-margin--0">
-          School Website
+          School website
```

Comment:
Confused by this change. `website`? We're building a website

Reply:
Are we building a website? What's a website? 🤣